src/util/std_expr.h

```diff
@@ -131,35 +115,31 @@ class symbol_exprt:public exprt
   }
 };
 
-/*! \brief Expression to hold a symbol (variable)
-*/
+/// Expression to hold a symbol (variable)
```

Comment:
Explain the difference to symbol_exprt

Reply:
`decorated_symbol_exprt` is actually not used. Shall I remove it?

Reply:
Please check with @kroening, I'd be all for removing it. A full revert of 5a2e5a834892 (will require conflict resolution) should do the trick.

Reply:
@tautschnig Thanks for the pointer to the original commit.